CHANGELOG.md

```diff
@@ -23,6 +23,8 @@ recommendations of [keepachangelog.com](http://keepachangelog.com/).
 - [#1285](https://github.com/paper-trail-gem/paper_trail/pull/1285) -
   For ActiveRecord >= 6.0, the `touch` callback will no longer create a new
   `Version` for skipped or ignored attributes.
+- Newly generated migrations won't generate a "{:null=>false}" column. Existing
```

Comment:
Should we specify that this is for Ruby 3?  Are you able to confirm this as well?